KarhooUISDK/CommonUI/VehicleCapacityView/NewVehicleCapacityView.swift

```diff
@@ -0,0 +1,133 @@
+//
+// Created by Bartlomiej Sopala on 10/03/2022.
+// Copyright (c) 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import KarhooSDK
+
+public struct KHNewVehicleCapacityViewID {
+    public static let capacityView = "vehicle_capacity_view"
+    public static let passengerStack = "passenger_stack"
+    public static let baggageStack = "baggage_stack"
+    public static let baggageIcon = "baggage_image"
+    public static let baggageCapacityLabel = "baggage_capacity_label"
+    public static let capacityIcon = "passenger_capacity_image"
+    public static let passengerCapacityLabel = "passenger_capacity_label"
+}
+
+final class NewVehicleCapacityView: UIStackView {
```

Comment:
Do we need prefix `New` on develop?

Reply:
I updated VehicleCapacityView to Matt's guides but then Diana told me to keep old design on QuoteCell to align with checkout view. So I saved changes as a new file and this NewVehicleCapacityView is currently unused, but ready to replace current layout in future, then we will delete VehicleCapacityView and rename this file.